### ISSUE TITLE
sequence_reader.slice() is skipping the first record in sequence file

### ISSUE THREAD

**Issue by thammegowda** [ContinuumIO/nutchpy#15]
Test case:

```python
import os
import nutchpy

path = os.path.dirname(nutchpy.__file__)
path = os.path.join(path,"ex_data", "crawldb_data")

first_slice = nutchpy.sequence_reader.slice(0, 2, path)
head = nutchpy.sequence_reader.head(2, path)
print "Head = %s\nSlice=%s" % (head[0][0], first_slice[0][0])
assert first_slice[0][0] == head[0][0]

```

**Comment by ahmadia** [comment]
Thanks for the report.

**Comment by chrismattmann** [comment]
fixed in #16 right?

**Comment by thammegowda** [comment]
@chrismattmann professor, not fixed in #16 . I suspect the [java code](https://github.com/ContinuumIO/nutchpy/blob/master/seqreader-app/src/main/java/com/continuumio/seqreaderapp/SequenceReader.java#L144) for this issue.

**Comment by chrismattmann** [comment]
no worries thanks @thammegowda I committed #17
